package.json

```diff
@@ -5,7 +5,7 @@
     "@commitlint/config-conventional": "^11.0.0",
     "ethlint": "^1.2.5",
     "husky": "^4.3.0",
-    "prettier": "^2.1.2",
+    "prettier": "^2.2.1",
```

Comment:
Can this... be in another PR? The change to the lock file is big

Reply:
this came automatically i think as a result of `npm i prettier` when trying to troubleshoot.